dotnet-client-libraries/ParsingHtmlWithAngleSharp/ParsingHtmlWithAngleSharp/FileEntry.cs

```diff
@@ -0,0 +1,54 @@
+﻿using AngleSharp.Io.Dom;
+
+namespace ParsingHtmlWithAngleSharp;
+
+public class FileEntry : IFile
+{
+    private readonly String _fileName;
+    private readonly Stream _content;
+    private readonly String _type;
+    private readonly DateTime _modified;
+
+    public FileEntry(String fileName, String type, Stream content)
+    {
+        _fileName = fileName;
+        _type = type;
+        _content = content;
+        _modified = DateTime.Now;
+    }
+
+    public Stream Body => _content;
+
+    public Boolean IsClosed => _content.CanRead == false;
```

Comment:
@vnikolaew 
Let's use the language keywords here. 
`bool` instead of `Boolean`, `string` instead of `String` and `int` instead of `Int32` e.t.c
There are other places in the code where this should also be applied.